index.css

```diff
@@ -0,0 +1,143 @@
+*{
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.Main-Page {
+  max-width: 500px;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height:max-width;
+  width: 375px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  border: 2px solid black;
+  padding: 120px, 24px, 120px, 24px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: #3c3a39;
+  width: 375px;
+  height: 92px;
+}
+
+.section {
+  height: 590px;
+  width: 375px;
+  background: black;
```

Comment:
- [ ] The background of the section is not shown. please add it to the CSS of the project.